test/suites/dev-tanssi-relay/inflation-rewards/test-rewards.ts

```diff
@@ -0,0 +1,134 @@
+import "@tanssi/api-augment";
+import { describeSuite, expect, beforeAll, DevModeContext } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+import { Header, ParaId, HeadData, Digest, DigestItem } from "@polkadot/types/interfaces";
+import { KeyringPair } from "@moonwall/util";
+import { fetchIssuance, filterRewardFromContainer, jumpToSession } from "util/block";
+import { PARACHAIN_BOND } from "util/constants";
+import { numberToHex, stringToHex } from "@polkadot/util";
+
+async function mockAndInsertHeadData(
```

Comment:
cocmment this function. BTW it looks very good

Reply:
Done in f117a242086e1c10409e64f069c79ec10d6b391c.